experimental/InvariantTheory/AlgebraicGroups.jl

```diff
@@ -0,0 +1,218 @@
+import AbstractAlgebra.Field
+import AbstractAlgebra.Generic.MatSpaceElem
+
+
+export AlgGroup, subgroup, base_field, ambient_ring, defining_ideal, presentation_matrix, AlgGroup_GL, AlgGroup_SL
+
+export LinearRepresentation, representation_on_sym_power, mapping_matrix, group
+
+
+@Markdown.doc """
+    AlgGroup{FieldType, RingType, RingElemType, MatrixType}
+
+An algebraic group G defined as an algebraic subvariety of the space GL(t,𝕜). 
+The information stored consists of 
+  * the base field 𝕜 of type `FieldType`;
+  * a polynomial ring `R` = 𝕜[x₁,…,xₙ] of type `RingType`, 
+    the ambient polynomial ring for G seen as an affine variety;
+  * an ideal `I` ⊂ `R` of type `MPolyIdeal{RingElemType}` defining G 
+    as a subscheme of 𝔸ⁿ(𝕜);
```

Comment:
I see that you are quite fond of using fancy Unicode in docstrings, but note that we do this nowhere else and so this is rather inconsistent.

Also, why is "G" written as a plain letter, not as ``` `G` ``` or ``` ``G`` ``` (the latter is math mode), while you do this for I and R?